apps/web/src/app/api/og/route.tsx

```diff
@@ -9,24 +12,48 @@ const size = {
 
 const TITLE = "Open Status";
 const DESCRIPTION = "An Open Source Alternative for your next Status Page";
+const LIMIT = 40;
 
 const interRegular = fetch(
   new URL("../../../public/fonts/Inter-Regular.ttf", import.meta.url),
 ).then((res) => res.arrayBuffer());
 
+const interLight = fetch(
+  new URL("../../../public/fonts/Inter-Light.ttf", import.meta.url),
+).then((res) => res.arrayBuffer());
+
 const calSemiBold = fetch(
   new URL("../../../public/fonts/CalSans-SemiBold.ttf", import.meta.url),
 ).then((res) => res.arrayBuffer());
 
 export async function GET(req: Request) {
   const interRegularData = await interRegular;
+  const interLightData = await interLight;
   const calSemiBoldData = await calSemiBold;
 
   const { searchParams } = new URL(req.url);
+
   const title = searchParams.has("title") ? searchParams.get("title") : TITLE;
   const description = searchParams.has("description")
     ? searchParams.get("description")
     : DESCRIPTION;
+  const siteId = searchParams.has("siteId")
+    ? searchParams.get("siteId")
```

Comment:
I'm a bit confused by siteID shouldn't get pageID or monitorId ?

Reply:
So in tinybird, we currently use `siteId` for the query. You are absolutely right: this should be `monitorId`. I'll change the search param to it. But we will need a refactor (and `version` change to update the params for tinybird - this should be a different PR).